### DISCUSSION TITLE
Brother label print suddenly stopped working

### DISCUSSION THREAD

**Discussion by martin323232** [inventree/InvenTree#5050] [Q&A] [answered]
Hi,

Have had inventree working great for 6 months now, but today when adding new parts there is an error printing out labels. 
Changing label rotation in plugin settings from 270  to 180 it will print but then output i printet in hight not width. 
No matter what dimension I set in label its always the same error

![image](https://github.com/inventree/InvenTree/assets/6439801/757ca941-20bb-4cbe-a8fa-084fed7ca1eb)
![image](https://github.com/inventree/InvenTree/assets/6439801/3e4232fd-ab38-473b-b07d-6b9c42356f6a)
![image](https://github.com/inventree/InvenTree/assets/6439801/7ca1082c-0f00-4961-836c-e787123e3a29)
![image](https://github.com/inventree/InvenTree/assets/6439801/339c6bf0-720f-4fd6-8ea5-58b1d6d744f9)

**Comment by matmair** [comment]
@martin323232 you are running an old version of InvenTree and the newest one of the brother plugin. Either downgrade the plugin to 0.6.0 or update InvenTree to 0.11.x
Your server probably restarted and updated the non-pinned plugins.

**Comment by martin323232** [comment]
@matmair updated server to latest dover version
![image](https://github.com/inventree/InvenTree/assets/6439801/0c2d36f4-30ef-4d53-9aa3-6c4895467b06)

Same issue with printing labels. But now I have an issue with Inventree.
Last time i updated everything worked fine after, now I only have the side menu bar on the index page.
All other pages are blank, except from part pages.
Is this a known issue?
What would procedure be to downgrade both inventree and plugin on Docker?

![image](https://github.com/inventree/InvenTree/assets/6439801/6bc0a650-4c34-4d43-873b-e789816fda0d)


![image](https://github.com/inventree/InvenTree/assets/6439801/0983fa8a-f80f-4623-852f-8c1dbe562dab)
![image](https://github.com/inventree/InvenTree/assets/6439801/486b1f5e-d840-4972-a0d2-d3212b35fc75)
![image](https://github.com/inventree/InvenTree/assets/6439801/31fc4b88-0288-4912-bbed-a654de8e4ec8)

**Reply by wolflu05** [reply]
Have you tried https://docs.inventree.org/en/latest/faq/#feature-x-does-not-work-after-update

**Reply by martin323232** [reply]
Hard-refreshing the browser cache worked for invensense, but label printing still have an issue. 
How can I downgrade the plugin?

**Reply by wolflu05** [reply]
Generally it's better to create an issue at the brother plugins repo. But habe you tried removing the plugin config and readding it? 
1. go to the admin panel
2. select plugin config Model
3. select the brother plugin config with the checkbox
4. select in the top drop-down delete and press the button next to it
5. restart InvenTree, enable the plugin again and enter your config

**Reply by matmair** [reply, marked as answer]
Is the issue still the same? You can downgrade the plugin by visiting the plugins.txt file and append the version you want at the end in pip annotation (eg. appending `==0.6.0`).

**Reply by martin323232** [reply]
@matmair thanks, that worked. 
btw. is there an option to set the Brother printer as default?
Now its set to "Export to PDF"
![image](https://github.com/inventree/InvenTree/assets/6439801/8807c51e-df10-4320-8982-5e3599d24585)

**Reply by matmair** [reply]
No

**Comment by SergeoLacruz** [comment]
I think it is here: 
https://github.com/inventree/InvenTree/pull/4362
